### ISSUE TITLE
i18n message differences for theme variants

### ISSUE THREAD

**Issue by adhurjaty** [keycloakify/keycloakify#643]
Hey,

I'm using the theme variants feature. It's working well for the most part, but there doesn't seem to be any way for us to specify different messages for different themes (such as headers having different contents based on your theme)

The hacky way we're working around this is by defining the identity function as `createUseI18n` so we can assign the translations to a variable while still being able to write to the messages_*.properties output
```
const createUseI18n = <T> (id: T) => id;

const translations = createUseI18n({
  en: {
    alphanumericalCharsOnly: "Only alphanumerical characters",
    "variant1:header": "Hello variant1",
    "variant2:header": "Hello variant2",
...
```
And aliasing the real function
```
import { createUseI18n as createUseI18nAlias } from "keycloakify/login";
```
To take in the `translations` object. Then we're finding all the keys with a `:` in them and then overriding the default `useI18n` function to check the `msgStr` key against the variant keys, then applying the theme name to the key to get the theme scoped translation.

Then, we are creating a postBuild step to clean up the messages_*.properties files.

This results in a lot of weird, hard to parse code for us.

Is there a way to maybe add another parameter to `createUseI18n` to accept theme variant overrides?

**Comment by garronej** [comment]
Hello @adhurjaty,  

Since it's the thrird time this has been requested I'll work something out to support it natively

**Comment by garronej** [comment]
Done: 
https://docs.keycloakify.dev/theme-variants#different-text-for-each-of-your-theme-variants  

https://docs.keycloakify.dev/faq-and-help/migration-guides/v10-greater-than-v11
